### PR TITLE
chore: add volunteer hrshtmlng

### PR DESCRIPTION
**Pull request by hrshtmlng**

### CODE REVIEW

File: data/users/hrshtmlng.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Harshit Malang",
+  "bio": "Im bridging the gap between design and code as a Frontend Developer",
```
Comment (codewithvoid) [inline data/users/hrshtmlng.json:3]: just mention the title here -> `Frontend Developer`

File: data/users/hrshtmlng.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "image": "https://twitter.com/hrshtmlng/photo",
+  "categories": [
+    {
+      "slug": "programming | software-development | web-development ",
```
Comment (codewithvoid) [inline data/users/hrshtmlng.json:7]: move langauges to programming category and can strip version from technologies. updated in below example
```json
[
  {
    "slug": "web-development",
    "topics": [
      "HTML",
      "CSS"
    ]
  },
  {
    "slug": "programming",
    "topics": [
      "JavaScript",
      "Python"
    ]
  }
]
```

File: data/users/hrshtmlng.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+      "topics": ["HTML6","CSS3","JavaScript","Python"]
+    }
+  ],
+  "contacts": {
+    "twitter": "https://twitter.com/hrshtmlng"
```
Comment (codewithvoid) [inline data/users/hrshtmlng.json:12]: just twitter handle here
```json
"twitter": "hrshtmlng"
```

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thanks for volunteering @hrshtmlng. Left some comments.

**Comment by hrshtmlng** [comment]
Done check it out again

File: data/users/hrshtmlng.json (hunk 2 above)
Comment (codewithvoid) [reply data/users/hrshtmlng.json:7]: Can you update this too. One array entry per category.

**Comment by codewithvoid** [comment]
@hrshtmlng can you please make the suggested changes. Thanks!

File: data/users/hrshtmlng.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Harshit Malang",
+  "bio": "Frontend Developer",
+  "image": "https://twitter.com/hrshtmlng/photo",
+  "categories": [
+    [
```
Comment (codewithvoid) [inline data/users/hrshtmlng.json:6]: remove nested array..just one `[` is required

File: data/users/hrshtmlng.json (hunk 3 above)
Comment (codewithvoid) [reply data/users/hrshtmlng.json:24]: resolve as per the comment

File: data/users/hrshtmlng.json [hunk 5]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Harshit Malang",
+  "bio": "Frontend Developer",
+  "image": "https://twitter.com/hrshtmlng/photo",
```
Comment (codewithvoid) [inline data/users/hrshtmlng.json:4]: this twitter photo url won't render..use github one instead 
https://avatars.githubusercontent.com/u/101478591?v=4

**Review by codewithvoid — CHANGES_REQUESTED** [review]

**Review by codewithvoid — APPROVED** [review]

**Comment by codewithvoid** [comment]
Thanks for making the changes. Welcome!

**Pull request merged**